porter/v1/cluster_control_plane.proto

```diff
@@ -740,6 +740,31 @@ message AppInstance {
   int64 porter_app_id = 4;
 }
 
+// CreateNotificationRequest is the request object for creating notifications
+message CreateNotificationRequest {
+  int64 project_id = 1;
+  int64 cluster_id = 2;
+  // deployment_target_identifier is the object that identifies the deployment target for the app
+  DeploymentTargetIdentifier deployment_target_identifier = 3;
+  // app_id is the id of the app
+  int64 app_id = 4;
```

Comment:
Can we not pass in app id and just search by app name? Would like to make it easier to switch to app instance id ~eventually